docs/apache-airflow-providers-databricks/operators/submit_run.rst

```diff
@@ -52,6 +52,8 @@ one named parameter for each top level parameter in the ``runs/submit`` endpoint
 
 All other parameters are optional, and described in the documentation of the ``DatabricksSubmitRunOperator`` class.
 
+Another way to do is use the param tasks to pass array of objects to instantiate this operator. Here the value of tasks param that is used to invoke ``api/2.1/jobs/runs/submit`` endpoint is passed through the ``tasks`` param in ``DatabricksSubmitRunOperator``. Instead of invoking single task, you can pass array of task and submit a one-time run.
+
 
```

Comment:
@eladkal @josh-fell This howto documentation already contains information about instantiating the operator. It's additionally present in docstring with example too. Could you tell me if we can move the example part from docstring and retain this information only in howto guide?

Reply:
In my opinion there is no need to duplicate information the operator can reference the docs (we do that in other operators as well)